crates/solidity/inputs/language/src/dsl.rs

```diff
@@ -0,0 +1,915 @@
+use codegen_grammar::*;
+
+slang_grammar! {
+
+    config = {
+
```

Comment:
I don’t see a hierarchy for spec sections/topics here. I wonder how are they specified in the new DLS?

Reply:
They aren't. The 'change-spec-to-use-DSL' task needs to add that, and to the `Grammar` (=> `LanguageSpec`) model.

Reply:
It is straightforward to add if the order didn't change, but now they are sorted alphabetically, it is a lot more work to manually group them again.

If you are generating this file via a script, is it possible to maintain the original ordering for now?

Reply:
I generated from the YAML once, from then on it's manual work e.g. lexical contexts, pushing the versioning down to the leaves. So I'm not keen on re-exporting and doing that again.

However, we can regenerate from the current form with grouping, if there were a way around it at the moment, or even if there were a temporary ancillary document with the required grouping.

Reply:
I'm very keen on having the spec be commitment-less in the sense that it is always easy to regenerate.